web/user/src/pages/video/[...id].vue

```diff
@@ -0,0 +1,297 @@
+const i18n = useI18n()
+
+const dt = (str: keyof I18n['videos']['details']) => {
+  return i18n.t(`videos.details.${str}`)
+}
+
+const router = useRouter()
+const route = useRoute()
+
+const isLoading = ref<boolean>(false)
+const video = ref<VideoResponse | undefined>(undefined)
+
+const comments = ref<VideoComment[]>([])
+const commentFormData = ref<string>('')
+const commentIsSending = ref<boolean>(false)
+
+const videoId = computed<string>(() => {
+  return route.params.id as string
+})
+
+const handleSubmitComment = async () => {
+  try {
+    commentIsSending.value = true
+    await postComment(videoId.value, commentFormData.value)
+    commentFormData.value = ''
+    const res = await getComments(videoId.value)
+    comments.value = res.comments
+  }
+  catch (e) {
+    snackbarItems.value.push({
+      text: 'コメントの送信に失敗しました。',
+      isShow: true,
+    })
+    console.log(e)
+  }
+  finally {
+    commentIsSending.value = false
+  }
+}
+
+const videoRef = ref<{ videoRef: HTMLVideoElement | null }>({ videoRef: null })
+
+const videoPlayerHeight = computed(() => {
+  if (videoRef.value.videoRef) {
+    if (videoRef.value.videoRef.offsetWidth >= 768) {
+      return 0
+    }
+    return videoRef.value.videoRef.offsetHeight
+  }
+  return 0
+})
+
+const selectedTab = ref<'product' | 'comment'>('product')
+
+const clickTab = (tab: 'product' | 'comment') => {
+  selectedTab.value = tab
+}
+
+const handleClickItem = (productId: string) => {
+  router.push(`/items/${productId}`)
+}
+
+const handleClickExperienceItem = (experienceId: string) => {
+  router.push(`/experiences/${experienceId}`)
+}
+
+const handleClickAddCart = (name: string, id: string, quantity: number) => {
+  const message = i18n.t('items.details.addCartSnackbarMessage', {
+    itemName: name,
+  })
+  addCart({ productId: id, quantity })
+  snackbarItems.value.push({
+    text: message,
+    isShow: true,
+  })
+}
+
+const handleCLickCoordinator = (id: string) => {
+  router.push(`/coordinator/${id}`)
+}
+
+const fetchComments = async () => {
+  const res = await getComments(videoId.value)
+  comments.value = res.comments
+}
+
+await useAsyncData(`schedule-${videoId.value}`, async () => {
+  isLoading.value = true
+  const videoRes = await getVideo(videoId.value)
+  video.value = videoRes
+  isLoading.value = false
+})
+
+onMounted(() => {
+  fetchComments()
+  const interval = setInterval(() => {
+    fetchComments()
+  }, 3000)
+
+  onUnmounted(() => {
+    clearInterval(interval)
+  })
+})
+
+useSeoMeta({
+  title: '動画',
+})
+</script>
+
+<template>
+  <template
+    v-for="(snackbarItem, i) in snackbarItems"
+    :key="i"
+  >
+    <the-snackbar
+      v-model:is-show="snackbarItem.isShow"
+      :text="snackbarItem.text"
+    />
+  </template>
+
+  <div
+    class="mx-auto grid max-w-[1440px] grid-flow-col auto-rows-max grid-cols-3 gap-8 text-main xl:px-14"
+  >
+    <template v-if="video">
+      <div class="col-span-3">
+        <the-video-player
+          ref="liveRef"
```

Comment:
先の`script`の‘videoRef`と紐づけるならここを変えると良さそう！

```suggestion
          ref="videoRef"
```